Src/Common/RequestResponseHeaders.cs

```diff
@@ -13,12 +13,22 @@ internal static class RequestResponseHeaders
         /// <summary>
         /// Source key in the request context header that is added by an application while making http requests and retrieved by the other application when processing incoming requests.
```

Comment:
Was the `Request-Context` introduced in separate PR? I do not remember it. Can you please send me a link